src/dash-table/DataTable.js

```diff
@@ -24,23 +25,64 @@ export default class DataTable extends Component {
     }
 
     render() {
+        if (!this.isValid()) {
+            Logger.error(`Invalid combination of filtering / sorting / pagination`);
+            return (<div>Invalid props combination</div>);
+        }
+
+        if (!this.allValidColumns()) {
+            Logger.error(`Invalid column format`);
+            return (<div>Invalid props combination</div>);
+        }
+
+        return this.props.id ? (<RealTable {...this.props} />) : (<RealTable {...this.props} id={this.getId()} />);
```

Comment:
Due to the way we use prop-types, function-based validations are not available to us -- making sure the column format props involving arrays contain the expected number of items -- if not, the table is deemed invalid